eslint.config.js

```diff
@@ -47,6 +48,10 @@ module.exports = [
 				}
 			],
 			"no-implicit-globals": "error",
+			"no-unused-vars": [
+				"error",
+				{ caughtErrorsIgnorePattern: "^e$" }
```

Comment:
I'd prefer to just use the same style as in other places, i.e. a prefix with `_`. But since it's just for `3.x-stable`, I'm fine with this version - as long as we don't plan to propagate it to `main`.

Reply:
I considered that, but I think our instinct is to use `e`, as was evident from about 90% of them. Because it's a separate rule, I think we can consider it separate from function parameters.

Reply:
What’s the reason this wasn’t needed on `main`?

BTW, if we’re special casing unused catch parameters in a different way than the rest of the code then I see no point; let’s just not check `catch` at all. What’s the point? It will always have one parameter, whether used or not. Checking `catch` gl for unused parameters will make sense in jQuery 5 where we’ll be able to use a parameterless `catch`.

If we remove `catch` parameter checking, we’ll be able to remove some modifications from this PR.

Reply:
I do see a point, although minor. It's simply to ensure that all of these are the same. Having `err` some places and `e` in others, which not a big deal, is something we can clean up with this rule.

Reply:
@timmywil but this is not a rule ensuring all catch parameters are named the same, only unused ones. I’d rather avoid such special one-off rules.

Reply:
Ensuring the unused ones are all the same is still useful imo.

Reply:
oh, and this wasn't needed on main because we can't upgrade eslint to v9 there yet.

Reply:
FWIW, I'd be more inclined to change this to use `_` than to remove the rule altogether.

Reply:
Both options work for me!

Reply:
Ok, done. Also, I've been switching branches quite often and found it annoying that 3.x always failed lint until I removed any built files from a main branch build. I've fixed that by adding them to ignore in eslint and .gitignore.

Reply:
> I've fixed that by adding them to ignore in eslint and .gitignore.

Sounds good to me. I’m not hitting this issue as I keep separate clones for the `main` & `3.x-stable` branches.